eth/watchers/topics.go

```diff
@@ -0,0 +1,23 @@
+package watchers
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+)
+
+var eventSignatures = []string{
+	"Unbond(address,address,uint256,uint256,uint256)",
+	"Rebond(address,address,uint256,uint256)",
+	"WithdrawStake(address,uint256,uint256,uint256)",
+	"NewRound(uint256 round, bytes32 blockHash)",
```

Comment:
The event topic hash omits parameter names and spaces so the event signature for `NewRound` should be:

```
NewRound(uint256,bytes32)
```

Reply:
woops, totally right. fixed in 80d3e25